ui/app/AppLayouts/Chat/views/communities/CommunityPermissionsView.qml

```diff
@@ -92,27 +87,26 @@ StatusScrollView {
                                    ? proxiedChannelsModel : communityItemModel
                 isPrivate: model.isPrivate
 
-                onEditClicked: root.editPermission(model.index, model.holdingsListModel,
-                                                   model.permissionsObjectModel,
-                                                   model.channelsListModel, model.isPrivate)
+                onEditClicked: root.editPermissionRequested(model.index)
+                onDuplicateClicked: root.duplicatePermissionRequested(model.index)
 
-                onDuplicateClicked: store.duplicatePermission(model.index)
-                onRemoveClicked: root.removePermission(model.index)
+                onRemoveClicked: {
+                    d.permissionIndexToRemove = index
+                    declineAllDialog.open()
+                }
             }
         }
     }
 
-    Component {
-        id: deletePopup
-        ConfirmationDialog {
-            id: declineAllDialog
-            header.title: qsTr("Sure you want to delete permission")
-            confirmationText: qsTr("If you delete this permission, any of your community members who rely on this permission will loose the access this permission gives them.")
-            onConfirmButtonClicked: {
-                store.removePermission(d.permissionIndexToRemove)
-                close()
-            }
+    ConfirmationDialog {
+        id: declineAllDialog
+
+        header.title: qsTr("Sure you want to delete permission")
+        confirmationText: qsTr("If you delete this permission, any of your community members who rely on this permission will loose the access this permission gives them.")
```

Comment:
s/loose/lose

Reply:
same as above, fixed and commented in figma